### PR TITLE
Move to LDK 0.0.114

### PR DESCRIPTION
**Pull request by benthecarman**

### CODE REVIEW

File: node-manager/src/node.rs [hunk 1]
```diff
@@ -923,12 +938,13 @@ pub(crate) fn create_peer_manager(
 
     PeerManagerImpl::new(
         lightning_msg_handler,
-        km.get_node_secret(Recipient::Node)
-            .expect("Failed to get node secret"),
+        // km.get_node_secret(Recipient::Node)
+        //     .expect("Failed to get node secret"),
```
Comment (TonyGiorgio) [inline node-manager/src/node.rs:942]:
```suggestion
```

**Review by TonyGiorgio — COMMENTED** [review]
minor nit, going to test now to see how this works in the app

File: node-manager/src/node.rs [hunk 2]
```diff
@@ -288,9 +275,11 @@ impl Node {
...
                     background_processor_logger.clone(),
                     Some(scorer.clone()),
-                    |d| async move {
-                        sleep(d.as_millis() as i32).await;
-                        false
+                    |d| {
+                        Box::pin(async move {
+                            sleep(d.as_millis() as i32).await;
+                            false
+                        })
```
Comment (TonyGiorgio) [inline node-manager/src/node.rs:282]: sometimes i see this error that I don't remember really seeing before: 

```
node_manager.js:1681 panicked at '`async fn` resumed after completion', node-manager/src/node.rs:279:34
```

Comment (benthecarman) [reply node-manager/src/node.rs:282]: yeah needed the Box::pin to make it compile, not sure if that was correct or not

Comment (TonyGiorgio) [reply node-manager/src/node.rs:308]: this was fixed with https://github.com/MutinyWallet/mutiny-node/pull/328/commits/10f2353fea9cb2071a7f14cec75f75230914bbc1 and LDK is merging it upstream

File: node-manager/Cargo.toml [hunk 3]
```diff
@@ -22,17 +22,21 @@ bitcoin-bech32 = "0.12"
 js-sys = "0.3.60"
 secp256k1 = "0.24.0"
 bitcoin_hashes = { version = "0.11", default-features = false }
-bitcoin = { version = "0.29.2", features = ["serde"] }
-bdk = { version = "0.27.0", default-features = false, features = ["keys-bip39", "esplora", "use-esplora-reqwest", "async-interface"] }
+bitcoin = { version = "0.29.2", default-features = false, features = ["serde", "secp-recovery"] }
+# TODO waiting for esplora version 0.4.0
+bdk = { git = "https://github.com/benthecarman/bdk", branch = "tmp", default-features = false, features = ["keys-bip39", "esplora", "use-esplora-reqwest", "async-interface"] }
```
Comment (benthecarman) [inline node-manager/Cargo.toml:27]: we can wait for bdk v0.27.2 if we want, should be any day now

**Review by benthecarman — COMMENTED** [review]
otherwise lgtm

**Pull request merged**